src/test/resources/pages/studentHomeHTMLEmpty.html

```diff
@@ -6,23 +6,28 @@
       </h1>
       <br/>
       <div class="overflow-auto alert alert-warning" id="statusMessage">
-         <div style="text-align:left;">
```

Comment:
@sujeet14108 I believe there are conflicts because you're modifying this file without syncing with the main repository's latest `master` branch. Two things ought to be done here:

1. Revert the changes in this file, sync with the latest `master` branch.
2. If you want to make modifications to HTML files, don't do it manually; instead use [God mode](https://github.com/TEAMMATES/teammates/blob/master/devdocs/godmode.md), a tool we have designed for modifying expected HTML files quickly.